docs/getting_started_jax.ipynb

```diff
@@ -0,0 +1,638 @@
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0kIyBosUoT_j"
+      },
+      "source": [
+        "## 2. Why Shardy?"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3EdCWagVNiuF"
+      },
+      "source": [
+        "### Readability\n",
+        "\n",
+        "As seen above, it's much easier to read the shardings and shard_maps and understand how they match what is happening in the JAX code. Similarly GSPMD propagation will give back HLO code - not MLIR which both Shardy and `jax.jit.lower` return."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "si1rYISKoYNO"
+      },
+      "source": [
+        "### Interpretability\n",
+        "\n",
+        "We are planning on exposing a feature we call \"user priorities\" (not in JAX yet!). It allows you to attach a value telling Shardy how important a tensor's dimension sharding is over other constraints in the program.\n",
+        "\n",
+        "Higher prioritied are defines as lower values (lowest being 0, think of it as a p0 priority tasks).\n",
+        "\n",
+        "```python\n",
+        "PartitionSpec(None, 'x', 'y', priorities=(None, 0, 1))\n",
+        "```\n",
+        "\n",
+        "Here the sharding of dim 1 on `x` has a higher priority than dim 2 on `y`, meaning dim 1 will be propagated through the program first and then dim 2, meaning any potential sharding conflicts will be explicitly avoided by having `x` propagated first.\n",
+        "\n",
+        "This can be helpful for debugging models as well by having you break down your sharding strategies to separate rounds of propagation in Shardy. For example:\n",
+        "\n",
+        "* Priority 0: data parallelism\n",
+        "* Priority 1: megatron\n",
+        "* Priority 2: ZeRO sharding"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cR9YOSVeUSyy"
+      },
+      "source": [
+        "## FAQS\n",
+        "\n",
+        "Below is a list of questions you may have on various JAX features and capabilities.\n",
+        "\n",
+        "### JAX Sharding types\n",
+        "\n",
+        "#### What about GSPMDSharding?\n",
+        "\n",
+        "`GSPMDSharding`  is closely tied to the C++/protobuf representation inside the XLA compiler. As such the type itself won't be supported.\n",
+        "\n",
+        "#### How about different logical device ID orderings?\n",
+        "\n",
+        "`GSPMDSharding`s were the answer for this, but instead (TBD) JAX will add a `device_ids` field to `NamedSharding`. And then a different `@mesh` value will be added to the `StableHLO` which saves the different device orderings. You'd now just need to give names to this type of sharding while you didn't before.\n",
```

Comment:
Talking to the JAX team, since this isn't finalized, they'd like us to remove `How about different logical device ID orderings` for now.